<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tuiomultiplexer</class>
 <widget class="QDialog" name="tuiomultiplexer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ludique's TUIO multiplexer, calibrator and gestures</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Input TUIO </string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>36</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port : </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isListening">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>63</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Are we Listening?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="port">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65000</number>
    </property>
    <property name="value">
     <number>3333</number>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>20</width>
     <height>681</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>341</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestures</string>
   </property>
   <widget class="QScrollArea" name="scrollGest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>421</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>304</width>
       <height>419</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="addGesture">
   <property name="geometry">
    <rect>
     <x>647</x>
     <y>27</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Gesture</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="minDistance">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>200.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>15</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestures min Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>50</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestures num zones</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numZones">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>43</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>105</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration (s)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="minDuration">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="maxDuration">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>291</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Point Calibration</string>
   </property>
   <widget class="QPushButton" name="pointCalStart">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Point Calibration</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="normCoordinates">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalize Coordinates</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Num of Points :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance from border :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numPointsSpin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="distanceFromBorder">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>70</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>0.300000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.005000000000000</double>
    </property>
    <property name="value">
     <double>0.050000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Frames for each point :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="framesPointBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>106</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Threshold : </string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minDistanceToUpdateBox">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>176</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="sendCalibrated">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Calibrated Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveCal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadCal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>420</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Output TUIO</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaContentTUIO">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>294</width>
       <height>249</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>112</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>400</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>How long to keep the object alive</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="objectAlive">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>0.990000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
